beacon-chain/rpc/prysm/v1alpha1/validator/construct_generic_block.go

```diff
@@ -26,6 +27,9 @@ func (vs *Server) constructGenericBeaconBlock(sBlk interfaces.SignedBeaconBlock,
 
 	switch sBlk.Version() {
 	case version.Deneb:
+		if blobsBundle == nil {
```

Comment:
not sure if this check is worth it here

Reply:
I think we should revert this check, `blobsBundle` is a passthrough, if anything, it should be checked in `constructDenebBlock`